### ISSUE TITLE
[Follow up to #21323] Changing tablet orientation will re-render cards in news feed incorrectly [Tablet]

### ISSUE THREAD

**Issue by Uni-verse** [brave/brave-browser#21868]
## Description 
Changing tablet orientation will re-render cards in news feed incorrectly. Doesn't happen on mobile. 

Follow up for https://github.com/brave/brave-browser/issues/21323

## Steps to reproduce 

   1. Clean install 1.37
   2. Enabled News, Show news
   3. Scroll through news feed
   4. Change tablet orientation, scroll back up slightly
   5. Change orientation back again, scroll up and down 

## Actual result 

https://user-images.githubusercontent.com/17885425/160018739-abde34f4-53c4-48c4-8ef8-a08dbccea78b.mp4

![Screen Shot 2022-03-24 at 2 53 51 PM](https://user-images.githubusercontent.com/17885425/160018909-4a8c08cb-3297-4077-9eae-5e4497951243.png)

## Expected result
Cards should be rendered correctly

## Issue reproduces how often 
Easily

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? 
- Can you reproduce this issue with the current Play Store Beta version? 
- Can you reproduce this issue with the current Play Store Nightly version?

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Samsung Galaxy Tab S7 
   - Android version: Android 11
   
## Brave version
```
1.37.105 Chromium: 100.0.4896.46 (Official Build) (32-bit) 
Revision	5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}

```
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by kjozwiak** [comment]
Above requires `1.38.98` or higher for `1.38.x` verification.

**Comment by Uni-verse** [comment]
Verified using `1.38.103, Chromium 100.0.4896.127` on `Samsung Galaxy Tab S7` running `Android 12`

- [ ] Cards are displayed correctly when changing orientation while scrolling through news feed.

Issues encountered - TBD

https://user-images.githubusercontent.com/17885425/164331972-d4fbee4f-b47b-4f08-9ce4-1bebb75f4071.mp4

**Comment by alexsafe** [comment]
Reopening as i see it happening again. Probably a regression. Now it only happens for the first tab after turning News on.

**Comment by kjozwiak** [comment]
Closing as we don't re-open issues that have already been shipped. @alexsafe please create a new issue as this one already has a PR associated with it and has been shipped via `1.38.x`.
